docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -79,8 +81,19 @@ Preparing For a Release
    This should be an "Improvement" in the current sprint, with "Release Flocker $VERSION" as the title, and it should be assigned to yourself.
    The issue does not need a design, so move the issue to the "Coding" state.
 
+#. If this is a maintenance release, announce on Zulip's Engineering > Maintenance Release stream that a maintenance release is in progress.
+
+   ::
+
+      @engineering I am releasing from release/flocker-0.3.2. Please don't land anything on that branch until the release is complete.
+
 #. Create a clean, local Flocker release branch with no modifications:
 
+   .. note::
```

Comment:
Someone is going to get this wrong sooner or later. Do we have an issue for "automate the whole release process"? I think Adam made one maybe?